server/integration/testsuite/src/test/java/org/infinispan/server/test/security/rest/RESTCertSecurityIT.java

```diff
@@ -136,13 +148,23 @@ private String keyAddressUnsecured(String key) {
                 + server.getRESTEndpoint().getContextPath() + "/default/" + key;
     }
 
+    private HttpResponse handleIOException(IOException e, int expectedCode) throws IOException {
```

Comment:
I did this because
`return handleIOException(...)`
was better than
`handleIOException(...); return null;`
and it's only a test :)